BEP1.md

```diff
@@ -1,51 +1,51 @@
 # BEP 1: Purpose and Guidelines
 
 
-- [BEP 1: Purpose and Guidelines](#bep-1--purpose-and-guidelines)
-  * [1.  What is BEP?](#1--what-is-bep)
-  * [2.  BEP Rationale](#2--bep-rationale)
-  * [3.  BEP Types](#3--bep-types)
-  * [4.  BEP Workflow](#4--bep-workflow)
-  * [5.  Reference](#5--reference)
-  * [6.  License](#6--license)
+- [BEP 1: Purpose and Guidelines](#bep-1-purpose-and-guidelines)
+  - [1.  What is BEP?](#1--what-is-bep)
+  - [2.  BEP Rationale](#2--bep-rationale)
+  - [3.  BEP Types](#3--bep-types)
+  - [4.  BEP Workflow](#4--bep-workflow)
+  - [5.  Reference](#5--reference)
+  - [6.  License](#6--license)
 
 
 ## 1.  What is BEP?
 
-BEP stands for Binance Chain Evolution Proposal. Each BEP will be a proposal document providing information to the Binance Chain/DEX community. The BEP should provide a concise technical specification of the feature or improvement and the rationale behind it. Each BEP proposer is responsible for building consensus within the community and documenting dissenting opinions. Each BEP has a unique index number.
+BEP stands for BNB Chain Evolution Proposal. Each BEP will be a proposal document providing information to the BNB Chain community, both BNB Beacon Chain and BNB Smart Chain. The BEP should provide a concise technical specification of the feature or improvement and the rationale behind it. Each BEP proposer is responsible for building consensus within the community and documenting dissenting opinions. Each BEP has a unique index number.
 
 ## 2.  BEP Rationale
 
-BEP is the primary mechanism for proposing new features, for collecting community technical input on an issue, and for documenting the design decisions that go into Binance Chain. Because the BEP are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
+BEP is the primary mechanism for proposing new features, for collecting community technical input on an issue, and for documenting the design decisions that go into BNB Chain. Because the BEP are maintained as text files in a versioned repository, their revision history is the historical record of the feature proposal.
 
-For Binance Chain contributors, BEPs are a convenient way to track the progress of their implementation. This will give end users a convenient way to know the current status of a given feature, function or improvement.
+For BNB Chain contributors, BEPs are a convenient way to track the progress of their implementation. This will give end users a convenient way to know the current status of a given feature, function or improvement.
```

Comment:
“BEPs are a convenient way to track the progress of their implementation.” ->" it is a convenient way to track the progress of their implementation by BEPs"

Reply:
"This will give end users a convenient way to know the current status of a given feature, function or improvement."->"It will help end users to know the status of a given feature, function or improvement"

Reply:
accepted and updated